hasher-matcher-actioner/hmalib/common/models/models_base.py

```diff
@@ -66,6 +68,24 @@ def get_dynamodb_type_key(type: str) -> str:
     def remove_content_key_prefix(key: str) -> str:
         return key[len(DynamoDBItem.CONTENT_KEY_PREFIX) :]
 
+    @classmethod
+    def set_to_dynamodb_attribute(cls, value: t.Set) -> t.Set:
+        if not value or len(value) == 0:
```

Comment:
nit: is this check not redundant?
`None` and empty set both evaluate to `False`